frameworks/Ruby/rails/benchmark_config.json

```diff
@@ -23,6 +23,28 @@
       "notes": "",
       "versus": "rack-puma-mri"
     },
+    "postgresql": {
+      "json_url": "/hello_world/json",
```

Comment:
You can remove the json and plaintext properties here as they're already tested

Reply:
done